tests/testthat/test-tornado.R

```diff
@@ -47,15 +52,44 @@ test_that("TornadoDataMapping features", {
 })
 
 test_that("Regular tornado plots with their options work properly", {
+  skip_on_cran()
+  skip_if_not_installed("vdiffr")
+  skip_if_not(.Platform$OS.type == "windows")
+  skip_if_not(getRversion() >= "4.1")
```

Comment:
I think we can remove `skip_if_not(.Platform$OS.type == "windows")`
When I said "skip on Linux" I had the Linux ValidR platform in mind. 
But having 2 other conditions (vdiffr and R>=4.1) is even better and should be enough in this case (and as soon as the ValidR-platform supports it - the tests should run there as well)

Reply:
Got it! Thanks for the clarification. I have reverted that change.